util/gen_coverage_report.ts

```diff
@@ -79,7 +79,7 @@ const processRaidbossCoverage = async (manifest: string, coverage: Coverage) =>
     const triggerSet = (await import(triggerFileName)).default as LooseTriggerSet;
 
     let timelineContents: string | undefined = undefined;
-    if (triggerSet.timelineFile) {
+    if (triggerSet.timelineFile !== undefined && triggerSet.timelineFile.length > 0) {
```

Comment:
```suggestion
    if (triggerSet.timelineFile !== undefined) {
```

I don't think any trigger set special cases the empty string here, and I think we shouldn't handle it specially.